test/demo/createToast.inttest.ts

```diff
@@ -0,0 +1,83 @@
+import 'jest';
+
+import {Application, Identity} from 'hadouken-js-adapter';
+
+import {NotificationOptions} from '../../src/client';
+
+import * as notifsRemote from './utils/notificationsRemoteExecution';
+import {isCenterShowing, getCardsByNotification, assertDOMMatches} from './utils/notificationCenterUtils';
+import {delay} from './utils/delay';
+import {getToastWindow, getToastCards} from './utils/toastUtils';
+import {createApp} from './utils/spawnRemote';
+import {assertNotificationStored} from './utils/storageRemote';
+
+const validOptions: NotificationOptions = {
+    body: 'Test Notification Body',
+    title: 'Test Notification Title'
+};
+
+const testManagerIdentity = {uuid: 'test-app', name: 'test-app'};
+
+describe('When calling createNotification', () => {
+    describe('With the notification center not showing', () => {
+        beforeAll(async () => {
+            // Hide the center to be sure we get toasts
+            if (await isCenterShowing()) {
+                await notifsRemote.toggleNotificationCenter(testManagerIdentity);
+            }
+        });
+
+        let testApp: Application;
+        let testWindowIdentity: Identity;
+        beforeEach(async () => {
+            testApp = await createApp(testManagerIdentity, {});
+            testWindowIdentity = await testApp.getWindow().then(w => w.identity);
+        });
+
+        afterEach(async () => {
+            await notifsRemote.clearAll(testWindowIdentity);
+            await testApp.quit();
+        });
+
+        test('A toast is shown for the notification', async () => {
+            const note = await notifsRemote.create(testWindowIdentity, validOptions);
+            await delay(100);
+            const toastWindow = await getToastWindow(testApp.identity, note.id);
+            expect(toastWindow).not.toBe(undefined);
```

Comment:
This is probably something that will be done more thoroughly in Toast/ToastManager tests but probably useful to have here aswell:

Toast windows are being tested if they exist and have DOM elements but not if they are visible though `isShowing`/`opacity`.

Reply:
Hmmm, do we ever really change the visibility of toasts? They just get created/destroyed right?